### PR TITLE
Bump Istio to 1.18

### PR DESCRIPTION
**Pull request by videlov**
**Description**

Changes proposed in this pull request:

- Bump Istio to 1.18

**Related issue(s)**

https://github.com/kyma-project/istio/issues/175

### CODE REVIEW

**Comment by barchw** [comment]
The new registry is `https://console.cloud.google.com/artifacts/docker/kyma-project/europe/prod/`
https://github.com/kyma-project/istio/blob/60823bd42d6bd80b192cfa78c753c5ad694ab7dd/internal/reconciliations/proxy/sidecars.go#L26

File: Makefile [hunk 1]
```diff
@@ -108,6 +108,7 @@ vet: ## Run go vet against code.
 
 .PHONY: test
 test: manifests generate fmt vet envtest ## Run tests.
+	go version
```
[the remark this answers is not in the document]
Comment (videlov) [reply Makefile:111]: No really, but does not hurt to know what go version is running, if you think it is better removing it, I do not have preference :)

[the remark this answers is not in the document]
Comment (videlov) [reply Makefile:111]: Actually we do not need it, if it fails we can always debug it. Will remove it.

**Review by triffer — APPROVED** [review]

**Pull request merged**